bundler-packages/runtime-rs/wasi-worker.mjs

```
@@ -1,63 +1,63 @@
-    }
-  },
-});
-
-const emnapiContext = getDefaultContext();
-
-const __rootDir = parse(process.cwd()).root;
+	self: globalThis,
+	require,
+	Worker,
+	importScripts: (f) => {
+		;(0, eval)(`${fs.readFileSync(f, 'utf8')}//# sourceURL=${f}`)
```

Biome (1.9.4)

[error] The comma operator is disallowed.

Its use is often confusing and obscures side effects.

(lint/style/noCommaOperator)